src/lib.rs

```diff
@@ -39,7 +39,7 @@
         unused_parens,
         unused_unsafe,
         unused_variables,
-        while_true)]
+        while_true))]
 // This list is select `allow` warnings
 #![deny(trivial_casts,
```

Comment:
does it also need to be checked on this one?

Reply:
Maybe, I don't know why they are separated ^^'

Reply:
Should I merge them?

Reply:
For me, its a matter of whatever is easiest for when we next need to update this list of warnings.  I did a dump of the warnings and put them in.  I don't remember the details, if the `allow`-defaulted ones were separate or not.